src/app/containers/Ad/Canonical/index.jsx

```diff
@@ -31,10 +31,18 @@ const AdContainer = styled.section`
   ${({ slotType }) => (slotType === 'mpu' ? mpuStyles : leaderboardStyles)}
 `;
 
-export const getBootsrapSrc = queryString => {
+export const getBootstrapSrc = (queryString, useLegacy = false) => {
+  const {
+    SIMORGH_ADS_SCRIPT_TEST,
+    SIMORGH_ADS_SCRIPT_LEGACY_TEST,
+    SIMORGH_ADS_SCRIPT_LIVE,
+    SIMORGH_ADS_SCRIPT_LEGACY_LIVE,
+  } = process.env;
   const useLiveSrc = isLive() || queryString.includes('ads-js-env=live');
```

Comment:
I'm still wondering what happens with this query string when used on the test environment if mozart isn't passing it through - do we ignore it on the server render and then use it on the client? Can that cause any weird behaviour?